debian/control

```diff
@@ -33,7 +33,7 @@ Architecture: all
 Depends: ${misc:Depends},
          adaptation-hybris (= ${binary:Version}),
          ofono,
-         phosh-config-ofono,
+	 ofono2mm,
```

Comment:
nitpick: should be aligned with spaces like the other dependencies

Reply:
Oh, did not notice that!